docs/hub/notifications.md

```diff
@@ -0,0 +1,32 @@
+# Notifications
+
+Notifications allow you to know when new activities (pull requests or discussions) happen on models, datasets and spaces belonging to users or organizations you are watching.
```

Comment:
```suggestion
Notifications allow you to know when new activities (Pull Requests or discussions) happen on repositories belonging to users or organizations you are watching.
```

Reply:
(nit) i think it's fine to keep "models, datasets and Spaces" to be more ML specific